modules/updating-eus-to-eus-upgrade-cli.adoc

```diff
@@ -0,0 +1,146 @@
+// Module included in the following assemblies:
+//
+// * updating/preparing-eus-eus-upgrade.adoc
+
+:_content-type: PROCEDURE
+[id="updating-eus-to-eus-upgrade-cli_{context}"]
+= EUS-to-EUS update using the CLI
+
+.Prerequisites
+
+* Update OpenShift CLI `oc` to the target version before each update.
+
+.Procedure
+
+. Upgrade any OLM Operators to versions that are compatible with both versions you are updating to.
+
+. Verify that all machine config pools display a status of `UPDATED` and that no machine config pool displays a status of `UPDATING`.
+To view the status of all machine config pools, run the following command:
++
+[source,terminal]
+----
+$ oc get mcp
+----
++
+.Example output
++
+Output is trimmed for clarity:
++
+[source,terminal]
+----
+NAME     CONFIG                                         	UPDATED   UPDATING
+master   rendered-master-ecbb9582781c1091e1c9f19d50cf836c       True  	  False
+worker   rendered-worker-00a3f0c68ae94e747193156b491553d5       True  	  False
+----
+
+. Change to the `eus-4.12` channel. Run the following command:
++
+[source,terminal]
+----
+$ oc adm upgrade channel eus-4.12
+----
++
+[NOTE]
+====
+The `oc adm upgrade channel` command is only present in 4.9 or later.
+
+If you receive an error message indicating that `eus-4.12` is not one of the
+available channels, this indicates that Red Hat is still rolling out 4.10 to 4.12 EUS upgrades.
+This rollout process generally takes 45-90 days starting at the GA date.
+====
++
+
+. Pause the machine config pools that you want to skip reboots on. Run the following commands:
++
+[NOTE]
+====
+You cannot pause the master pool.
+====
++
+[source,terminal]
+----
+$ oc patch mcp/worker --type merge --patch '{"spec":{"paused":true}}'
+----
+
+. Update to version 4.11. Run the following command:
```

Comment:
actually, I believe it may be referred as Upgrade rather than update
```
❯ oc adm upgrade --help
Check on upgrade status or upgrade the cluster to a newer version
...
```
and in this doc
```this indicates that Red Hat is still rolling out 4.10 to 4.12 EUS upgrades```
as well as in [existing doc](https://docs.openshift.com/container-platform/4.12/updating/understanding-upgrade-channels-release.html):
```eus-4.y (only offered for EUS versions and meant to facilitate upgrades between EUS versions)```

however still, in many places in the doc, it is still referred as update. raising the awareness for the inconsistency with the intend of spawning a discussion on the topic.
apologies for the bikeshedding in advance :)
cc @wking @shellyyang1989

Reply:
Would that affect lines such as: 
Verify that all machine config pools display a status of `UPDATED` and that no machine config pool displays a status of `UPDATING`.

Should this be reworded to UPGRADE? If I remember right, the console still shows UPDATE

Reply:
For now, I've changed all "update" terms to "Upgrade", unless it's referring to a display of the CLI or web console. For the scope of this story, I think that should suffice. What do you think @evakhoni @shellyyang1989 @LalatenduMohanty ?

Reply:
thanks Olivia! my intention was actually to create a discussion on the topic of whatever such change is desired, since while we internally do refer to it as an Upgrade rather than an Update, however in the official documentation, as far as I've seen it mentioned interchangeably, so my intention is to spawn a discussion of whatever we desire to progress to more uniform reference, or rather proceed with the status quo, considering the quite large amount of docs that will require a change if we decide to progress in that direction. 
all said above however, is only when discussing the whole process of upgrading a cluster from one version to another, while as for the console output for the pools, oc client mentions `UPDATED   UPDATING   DEGRADED` ...

Reply:
You've brought up a great point @evakhoni :) 
After speaking with some of the other tech writers, it looks like we're trying to use the term "update" rather than "upgrade" when referring to updating the cluster to a new version. Here's the writing glossary to show what I'm talking about: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#u.

For the sake of this story, I think it makes sense to use "Update" when possible. What do you think? :)

Reply:
tbh, I'm pretty agonistic if only slightly inclined towards "upgrade" in the key terms. either is fine eventually, as long as the terminology matches across documentation to reduce customer confusion. also, I would like to hear some Dev opinion on the subject, in regards of the cluster outputs if via cli, webUI or api calls, compared to documentation terminology. with that said, it is not a showstopper anyway.

Reply:
I've put in https://issues.redhat.com/browse/OTA-932 proposing that we deprecate `oc adm upgrade` in favor of `oc adm update` so that hopefully in 4.14 we can update docs to use the new command. There's potential that we'd never remove the old command but it should emit an informational message that it's been deprecated.

Reply:
@sdodson +1 for taking care of consistency. whatever the term chosen, it's a good idea to use only one instead of confusing customers with interchangeable terms.